style.css

```diff
@@ -505,6 +599,31 @@ footer {
 
 /* Medium devices (landscape tablets, 768px and up) */
 @media only screen and (min-width: 768px) {
+  #modal {
+    padding-top: 4rem;
+    margin: 1.25rem auto;
+  }
+
+  #modal-close {
+    top: 1rem;
+    right: 1rem;
+    background-color: var(--glass);
+  }
+
+  .modal-title {
+    grid-area: 2 / span 1;
+  }
+
+  #modal-buttons-container {
+    grid-area: 2;
+    justify-content: flex-end;
+  }
+
+  .model-button {
+    font-size: 1.1rem;
+    gap: 1rem;
+  }
+
```

Comment:
[OPTIONAL] Well done on the popup 🎉 but in order to make it look better and enhance the UX, consider adding some margins. 😉

Reply:
Hi @didierganthier, Thanks for your kind suggestions:
> [OPTIONAL] Well done on the popup 🎉 but in order to make it look better and enhance the UX, consider adding some margins. 😉

Can you please specify to which element should I add margin?

Reply:
@BahirHakimy you should add margins to the popup modal, it's currently taking the whole screen so it might not be clear to users that they're seeing a popup. Take a look at the design and you'll see.

Reply:
> @BahirHakimy you should add margins to the popup modal, it's currently taking the whole screen so it might not be clear to users that they're seeing a popup. Take a look at the design and you'll see.
@didierganthier 
Sorry, but can you please add a screenshot?
As I see in my screen it has margins around it.
![Capture2](https://user-images.githubusercontent.com/73453971/216641982-2ef6b67f-39cb-491f-90e5-22b4fe8077e5.PNG)

![Capture](https://user-images.githubusercontent.com/73453971/216641943-823ab68e-b2bf-4f83-af27-6b31de62d336.PNG)

Reply:
It looks good in this 🎉 and I admire your desire to improve it even more. That's really rare. You're doing great man. 🙂

Reply:
Thank you very much @didierganthier
I really appreciate your advice and feedback.